packages/react-components/src/components/Switch/Switch.tsx

```diff
@@ -1,19 +1,25 @@
 import * as React from 'react';
 import cx from 'clsx';
-import { LockBlack as LockIcon } from '@livechat/design-system-icons/react/material';
+import {
+  LockBlack as LockIcon,
+  Check as CheckIcon,
+  Close as CloseIcon,
+} from '@livechat/design-system-icons/react/material';
 
 import { Icon, IconSize } from '../../components/Icon';
 import noop from '../../utils/noop';
 
 import styles from './Switch.module.scss';
+import { Loader } from '../Loader';
 
-const baseClass = 'switch';
+export const baseClass = 'switch';
 
-export type SwitchSize = 'basic' | 'compact';
+export type SwitchSize = 'large' | 'medium' | 'compact';
```

Comment:
From my perspective `Size` should be a common interface that is inherited by multiple components. However, this is not the time to change this in this PR

Reply:
Actually there was a discussion about it, and it should be fixed in the future (the issue should be in issues tab)

Reply:
Can we sort it ascending?